src/main/java/org/zeromq/ZMsg.java

```diff
@@ -263,7 +287,13 @@ public static ZMsg recvMsg(Socket socket, int flag)
   @Draft
   public static void recvMsg(ZMQ.Socket socket, int flags, Consumer<ZMsg> handler)
   {
-    handler.accept(ZMsg.recvMsg(socket, flags));
+    ZMsg msg = null;
+    try {
+      msg = ZMsg.recvMsg(socket, flags);
+    }
+    catch (ZMQException ignore) {
+    }
```

Comment:
I think if no exception handler is provided, we should just let the exceptions happen so that calling code can handle them.

Reply:
you're right, my mistake